### ISSUE TITLE
Files in .zip not stored as intended but with backslashes

### ISSUE THREAD

**Issue by besendorf** [botherder/androidqf#4]
The files in the .zip files are not stored as intended in subfolders but rather have `\` in their filenames. 
Example:
`\apks\com.google.android.wifi.resources.apk`
Seems to be a problem with Windows path handling

As a result mvt-android won't find the android-qf files because it looks for example for `*/syslog.txt`

Edit: I can't reproduce this issue on my Windows VM. First noticed it on a Windows Laptop. Bit confused. Will test some more.

**Comment by Te-k** [comment]
Hi @besendorf , I am interested if you can reproduce that bug, could you let me know?

**Comment by besendorf** [comment]
> Hi @besendorf , I am interested if you can reproduce that bug, could you let me know?

Yeah, I have a Windows Laptop that produces those .zip files. However the files only look broken on when opened on Linux. On Windows they look fine.
I found this thread on superuser which describes the problem. Maybe it is an issue with the zip library used by androidqf.
https://superuser.com/questions/1382839/zip-files-expand-with-backslashes-on-linux-no-subdirectories
